### ISSUE TITLE
Accessing remote encrypted data?

### ISSUE THREAD

**Issue by johntrandall** [jyruzicka/rubyfocus#7]
Was reading of encrypted data from the OmniSync server ever implamented? 

I see that there are #encrypted? methods, but I don't see any code doing the decryption to make the remote data readable. As a result (I think), when trying to access remove data I'm consistently getting the "Rubyfocus::OSSFetcherError, "Malformed OmniFocus zip file #{fq_zipfile}" error.

It looks like it is not trivial to decrypt the files. See https://discourse.omnigroup.com/t/omnifocus-sync-encryption-gory-technical-details/24611 TL;DR: PBKDF2-SHA256, AESWRAP, AES128-CTR, HMAC-SHA256....

**Comment by jyruzicka** [comment]
Nah, I had a go at it, but it very rapidly got beyond my scope. Definitely found it to be non-trivial - no libraries or the like that will do it.

**Comment by timwis** [comment]
I was able to decrypt it using [this python script from omni](https://github.com/omnigroup/OmniGroup/blob/master/Frameworks/OmniFileStore/DecryptionExample.py)
